frontend/taipy-gui/src/utils/formatConversion.ts

```diff
@@ -20,12 +20,12 @@ const re = {
     placeholder: /^\x25?(?:\.(\d+))?([b-giostuvxX])/, // Matches placeholders
 };
 
-/**
- * Parses a sprintf format string and returns a parse tree.
- * @param fmt The sprintf format string.
- * @returns The parse tree representing the structure of the format string.
- */
-const sprintf_parse = (fmt: string) => {
+const precisionFormat = (precision?: string, specifier?: string): string => {
+    // Default to precision of 2 if not specified
+    return "." + (precision?.slice(1) ?? "2") + specifier;
+}
+
+const sprintfParse = (fmt?: string): (string | { placeholder: string; })[] => {
```

Comment:
I don't know that I would have removed the comment alltogether.
It could be useful in the future...
Why not simply replate /** by /*?

Reply:
resolved